src/main/java/edu/ksu/canvas/requestOptions/ListRolesOptions.java

```diff
@@ -0,0 +1,26 @@
+package edu.ksu.canvas.requestOptions;
+
+/**
+ * A class to represent parameters to be passed to the "List the roles available to an account" API call.
+ * See <a href="https://canvas.instructure.com/doc/api/roles.html#method.role_overrides.api_index">
+ *              https://canvas.instructure.com/doc/api/roles.html#method.role_overrides.api_index</a>
+ */
+public class ListRolesOptions extends BaseOptions {
```

Comment:
This is an appropriate use of an options class for the `show_inherited` parameter. For bonus points, you could also implement the optional `state[]` parameter. This would involve creating an enum for the allowed values (there are only two in this case) and then accepting a list of that enum type. There are several similar methods already implemented. Like the `GetSingleCourseOptions.includes` method.